extensions/blocks/podcast-player/style.scss

```diff
@@ -1,11 +1,134 @@
 /**
- * Frontend (site) styles for Podcast Episodes
+ * Podcast Player block shared styles (editor & front-end).
  */
 
+$episode-v-padding: 15px;
+$episode-h-padding: 10px;
+$episode-status-icon-size: 22px;
+$text-color: #92959a;
+$text-color-hover: #000;
+$text-color-active: $text-color-hover;
+$text-color-error: #db5051;
+$block-bg-color: #fff;
+$block-border-color: $text-color;
+$font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;
+$font-size: 16px;
+
+/**
+ * Player's (block) parent element
+ */
 .wp-block-jetpack-podcast-player {
-	border: 1px solid;
+	border: 1px solid $text-color;
+	background-color: $block-bg-color;
+
+	// Player's state classes added to this element:
+	// &.is-playing {} // When audio starts playing
+	// &.is-paused {}  // When audio is paused
+	// &.is-error {}   // When playback error occured
+
+	audio {
+		display: none;
+	}
+}
+
+.podcast-player__episodes {
+	list-style-type: none;
+	display: flex;
+	flex-direction: column;
+
+	margin: 0;
+	padding: $episode-v-padding 0;
 }
 
-.wp-block-jetpack-podcast-player audio {
+.podcast-player__episode {
+	margin: 0;
+	color: $text-color;
+
+	&:hover {
+		color: $text-color-hover;
+	}
+
+	/**
+     * When episode "is-active", it means that it's been clicked by a user to
+     * start playback. Combine this class with the Player's state classes (see
+     * above) to apply styling for different scenarios.
+     */
+	&.is-active {
+		color: $text-color-hover;
+		font-weight: bold;
+	}
+}
+
+.podcast-player__episode-link {
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: space-between;
+	padding: $episode-h-padding $episode-v-padding;
+	font-family: $font-family;
+	font-size: $font-size;
+
+	text-decoration: none;
+	color: inherit;
+
+	.podcast-player__episode.is-active & {
+		.wp-block-jetpack-podcast-player.is-error {
+			padding-bottom: 0; // Make space for the error element that will be appended.
+		}
+	}
+}
+
+.podcast-player__episode-status-icon {
+	width: $episode-status-icon-size;
+
+	svg {
+		width: $episode-status-icon-size;
+		height: $episode-status-icon-size;
+	}
+
+	.podcast-player__episode.is-active & {
+		.wp-block-jetpack-podcast-player.is-paused & {
+			fill: none; // Currently there's no icon for paused state.
+		}
+		.wp-block-jetpack-podcast-player.is-playing & {
+			fill: $text-color-active;
+		}
+		.wp-block-jetpack-podcast-player.is-error & {
+			fill: $text-color-error;
```

Comment:
@jeryj, this doesn't seem to be working with the SVG references. I tried a bunch of ways to fix it but I couldn't get the colors to render properly, so I'm leaving this one for a follow-up PR. 
cc @marekhrabe

Reply:
This works fine in Firefox and Safari. Chrome doesn't like it. 🤔

Reply:
That's unusual! 😄

Reply:
This is fixed now. Here's what was wrong (in 2020 theme). Our structure looks like:

```
<svg><use xlink="#sound-icon /></svg>
<svg><defs><symbol id="#sound-icon"><path ... /></symbol></defs></svg>
```

The `<use>` part there ends up being a shadow-dom from the `<symbol>`. That’s all normal. But the global `path` CSS that 2020 uses gets applied to the SVG _definition list_.

Then when Chrome is using the shadow dom, that `path` CSS is applied in the SVG definition carries over to the `<use>`, even though it doesn’t show up in the dev tools. So some kind of render bug.

The fix is to add a class to the `<svg>` definitions:  `<svg class="podcast-player-icons"><defs><symbol id="#sound-icon"><path ... /></symbol></defs></svg>`

```
.podcast-player-icons {
 transition: none;
}
```